createrepo_c.spec

```diff
@@ -25,6 +25,12 @@
 %bcond_without zchunk
 %endif
 
+%if 0%{?rhel} || 0%{?fedora} < 30
```

Comment:
It's going to make its way into stable now: https://bodhi.fedoraproject.org/updates/FEDORA-2019-05908109f7